web/massastation/src/pages/Search/Search.tsx

```diff
@@ -8,6 +8,12 @@ import Upload from '../../components/SidePanels/Upload/Upload';
 import DomainModelItem from './DomainModelItem/DomainModelItem';
 import { routeFor } from '../../utils';
 
+// import { AccountObject } from '../../models/AccountModel';
+// import { useAccountStore } from '../../store/store';
+// import { URL } from '../../const/url/url';
+
+// import { Dropdown, Identicon } from '@massalabs/react-ui-kit';
+
 const FAVORITES = ['flappy', 'psychedelic', 'flappynathana1'];
```

Comment:
add these values in a unique source of truth like massaConstants.ts

Reply:
> add these values in a unique source of truth like massaConstants.ts

In which folder can I create this file ?

Reply:
it is already created is src/utils